README.md

```diff
@@ -133,16 +133,16 @@ check the general status of an interface:
         "NCP:HardwareAddress" => [F1D92A82C8D8FE43]
     ]
 
-Here we see that the NCP is in the `offline` state, along with a few
-additional bits of information like the version of the NCP and it's
+Here we see that the NCP is in the `offline` state along with a few
+additional bits of information, such as the version of the NCP and its
 hardware address. From here we can easily form a new network:
 
     $ sudo wpanctl form "wpantund-testnet"
     Forming WPAN "wpantund-testnet" as node type router
     Successfully formed!
     $
 
-And now if we check the status, we will see a lot more information:
+Now if we check the status, we will see more information:
```

Comment:
These habits are hard to kill. You write the way you talk. Only American English has a disconnect between written and spoken versions. We are moving closer to the spoken style, but we aren't there yet. Doing away with adverbs is a good basic rule (and then putting them back in exceptional cases only). Online grammar checkers, such as https://www.grammarly.com/ and http://www.hemingwayapp.com/, flag all adverbs. They run a point score on your doc, and you lose points with adverbs.